docs/user/openstack/bind-vfio-pci-kernel-module.md

```diff
@@ -0,0 +1,144 @@
+# Binding the vfio-pci kernel module to vhostuser port.
+
+When a VM is attached to a port of type dpdkvhostuserclient, as is the case when vms are
+spun up inside an OpenStack deployment configured for [ovs-dpdk][1] it is possible to use a pmd driver like the [VFIO
+driver][1] on the port.
+Before the interface can be used by a pmd driver,  the vfio-pci kernel module needs to be loaded and bound to
+the desired interface of the vm.
+
+In a regular VM running linux OS, the cli commands would look something like:
+```
+modprobe vfio enable_unsafe_noiommu_mode=1
+modprobe vfio-pci
+echo "0000:00:07.0" > "/sys/bus/pci/drivers/virtio-pci/unbind"
```

Comment:
Add a note on how to figure out the bus ID (lspci or lshw).

Reply:
I tend to use `grep PCI_SLOT_NAME /sys/class/net/*/device/uevent`, but you can also use `ethtool -i $ETH`

Later: oh, [here's](https://askubuntu.com/questions/654820/) a useful SO answer to maybe link to